x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/CrossClusterAccessSubjectInfo.java

```diff
@@ -36,29 +36,31 @@
 import java.util.Objects;
 import java.util.Set;
 
-public final class RemoteAccessAuthentication {
-    public static final String REMOTE_ACCESS_AUTHENTICATION_HEADER_KEY = "_remote_access_authentication";
+public final class CrossClusterAccessSubjectInfo {
```

Comment:
Going with subject info to remove the ambiguity of `cross_cluster_access` Authentication and `CrossClusterAccessAuthentication`